src/java.base/share/classes/java/net/HttpURLConnection.java

```diff
@@ -277,9 +277,11 @@ public void setFixedLengthStreamingMode(long contentLength) {
      * <p>
      * This method must be called before the URLConnection is connected.
      *
-     * @param   chunklen The number of bytes to write in each chunk.
-     *          If chunklen is less than or equal to zero, a default
-     *          value will be used.
+     * @param   chunklen The number of bytes to be written on the wire in
+     *          each chunk. This includes a chunk size header given as a
+     *          hexidecimal string (minimum of 1 byte), two CRLF's (4 bytes)
+     *          and a minimum payload length of 1 byte. If chunklen is less
+     *          than or equal to 5, a default value will be used.
      *
```

Comment:
I'd suggest something like this:

chunklen The number of bytes to be written in
each chunk, including the chunk header as a
hexadecimal string (minimum of 1 byte), two CRLF's (4 bytes)
and a minimum payload length of 1 byte. If chunklen is less
than or equal to 5, a higher default value will be used.

Reply:
Thanks @Michael-Mc-Mahon, I updated the text with respect to your suggestion there (which was definitely a bit more cleared than mine) and corrected my incorrect spelling of hexadecimal in my most recent changes just now.